app/templates/filters-dialog-content.hbs

```diff
@@ -0,0 +1,5 @@
+{{filters-dialog-content
+  model=model
```

Comment:
Maybe better to explicitly passing the `columns` and `componentName` instead of some obscure `model`?